gridstatus/ieso.py

```diff
@@ -0,0 +1,302 @@
+    # no repeated or missing hours in the raw data and we can safely use tz_localize
+    # without setting ambiguous or nonexistent times.
+    # https://www.ieso.ca/-/media/Files/IESO/Document-Library/engage/ca/ca-Introduction-to-the-Capacity-Auction.ashx
+    default_timezone = "EST"
+
+    status_homepage = "https://www.ieso.ca/en/Power-Data"
+
+    @support_date_range(frequency="YEAR_START")
+    def get_load(self, date, end=None, verbose=False):
+        """
+        Get hourly load by zone for a given date or from date to end date.
+
+        Args:
+            date (datetime.date): date to get load for. If end is None, returns
+                only data for this date.
+            end (datetime.date, optional): end date. Defaults to None. If provided,
+                returns data from date to end date.
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pd.DataFrame: zonal load as a wide table with columns for each zone
+        """
+        today = self._today()
+        date = self._handle_input_date(date)
+
+        date_only = date.date()
+
+        if date_only < pd.to_datetime(EARLIEST_ACTUAL_DEMAND).date():
+            raise NotSupported(
+                f"Load is not available before {EARLIEST_ACTUAL_DEMAND}.",
+            )
+
+        if date_only > today:
+            raise NotSupported("Load is not available for future dates.")
+
+        year = date.year
+        url = ACTUAL_DEMAND_TEMPLATE_URL.replace("YYYY", str(year))
+
+        r = self._request(url, verbose)
+
+        # Create a dataframe from the CSV file, skipping lines starting with \\
+        df = pd.read_csv(io.StringIO(r.text), comment="\\")
+
+        df["Interval Start"] = (
+            pd.to_datetime(df["Date"]) + pd.to_timedelta(df["Hour"], unit="h")
+        ).dt.tz_localize(self.default_timezone)
+
+        df["Interval End"] = df["Interval Start"] + INTERVAL_DURATION
+
+        df = utils.move_cols_to_front(df, ["Interval Start", "Interval End"])
+        cols_to_drop = ["Date", "Hour", "Diff"]
+        df = df.drop(cols_to_drop, axis=1)
+
+        col_mapper = {
+            col: f"{col} Load"
+            for col in df.columns
+            if col not in ["Interval Start", "Interval End"]
+        }
+
+        df = df.rename(columns=col_mapper)
+
+        # If no end is provided, return data from single date
+        if not end:
+            return df[df["Interval Start"].dt.date == date_only].reset_index(drop=True)
+
+        end_date_only = pd.to_datetime(end).date()
+
+        return df[
+            (df["Interval Start"].dt.date >= date_only)
+            & (df["Interval End"].dt.date <= end_date_only)
+        ].reset_index(drop=True)
+
+    @support_date_range(frequency="DAY_START")
+    def get_load_forecast(self, date, end=None, verbose=False):
+        """
+        Get forecasted load by forecast zone (Ontario, East, West) for a given date
```

Comment:
Note that these "zones" are different from the `get_load` zones. For the forecast, there are only three "zones": Ontario, East, and West. For the load, there are 11 zones: Ontario Demand, Northwest, Northeast, Ottawa, East, Toronto, Essa, Bruce, Southwest, Niagara, West. 

I don't think we should use zones in both places to avoid confusion. Perhaps for the load forecast, we can use "region?"